python/grass/temporal/abstract_space_time_dataset.py

```diff
@@ -1865,7 +1865,7 @@ def get_registered_maps(
             try:
                 dbif.execute(sql, mapset=self.base.mapset)
                 rows = dbif.fetchall(mapset=self.base.mapset)
-            except:
+            except Exception:
```

Comment:
A search in the code base shows this is used in grass.pygrass but not grass.temporal:

```bash
$ grep -IrnE DBError python/grass/
python/grass/exceptions/__init__.py:7:class DBError(Exception):
python/grass/pygrass/vector/table.py:16:from grass.exceptions import DBError
python/grass/pygrass/vector/table.py:543:        grass.exceptions.DBError: SQLite does not support to cast columns.
python/grass/pygrass/vector/table.py:568:            raise DBError("SQLite does not support to cast columns.")
```

What it should do, that depends on SQLDatabaseInterfaceConnection. It seems that it mostly let's exceptions through, so it might be the underlying database package exceptions.

Reply:
Depending on which database library is imported, ```sqlit3.Error``` or ```psycopg2.Error``` could be used to catch all exceptions. In this case, I would choose Exception, unless specific code is suggested, since this requires information about the imported package (sqlite3 or psycopg2) or the database connection string to identify database.  

There are more cases similar to this and I commented "Could grass.exceptions.DBError be used?" for them previously. I will keep ```Exception``` unless specific code is provided for them as well.

```python
exception Exception:
```

Reply:
Let's leave this for later and go just with exception.